libnvme-rs/src/nvme_uri.rs

```diff
@@ -42,12 +42,14 @@ impl Drop for NvmeStringWrapper {
 #[derive(Debug, PartialEq)]
 enum NvmeTransportType {
     Tcp,
+    Rdma,
 }
 
 impl NvmeTransportType {
     fn to_str(&self) -> &str {
         match self {
             NvmeTransportType::Tcp => "tcp",
+            NvmeTransportType::Rdma => "rdma",
```

Comment:
This is only used by tests so assuming `rdma` for connect upon seeing NvmeTransportType::Rdma , instead of fallback behaviour handling.